### PR TITLE
Improve mark-up, add owner name to domains list

### PR DESCRIPTION
**Pull request by yeremeyev**
https://sourceclear.atlassian.net/browse/FLAT-643

### CODE REVIEW

File: assets/javascripts/discourse/templates/csp-reports.hbs [hunk 1]
```diff
@@ -70,9 +43,50 @@
...
         </tr>
       {{/each}}
     </table>
-  {{/if}}
+  </section>
+
+  <section class="shared-domains mb+">
+    <h4>Shared with me</h4>
+
+    {{#if model.friend_domains}}
```
Comment (ArthurZaharov) [inline assets/javascripts/discourse/templates/csp-reports.hbs]: Please move this before shared-domains section tag.

File: assets/javascripts/discourse/templates/csp-reports.hbs [hunk 2]
```diff
@@ -1,64 +1,37 @@
...
-          <button {{action "setNotificationType" "weekly"}} {{bind-attr class=":btn--xs domain.model.isWeekly:btn--default:btn"}}>Weekly</button>
-          <button {{action "setNotificationType" "monthly"}} {{bind-attr class=":btn--xs domain.model.isMonthly:btn--default:btn"}}>Monthly</button>
-        </td>
-        <td>
-          {{d-button action="destroy" icon="trash-o"}}
-        </td>
-      </tr>
-    {{/each}}
-  </table>
+  <section class="my-domains mb+">
```
Comment (ArthurZaharov) [inline assets/javascripts/discourse/templates/csp-reports.hbs]: Can you add check to this section

```
{{#if model.domains}}
```

**Comment by yeremeyev** [comment]
@ArthurZaharov thanks for a comments. Can you take a look again?

**Comment by ArthurZaharov** [comment]
:+1:

**Pull request merged**